Documentation/platforms/container-linux-systemd.md

```diff
@@ -0,0 +1,138 @@
+## Introduction
+
+The following guide shows how to run etcd with [systemd][systemd-docs] under [Container Linux][container-linux-docs].
+
+## etcd v0, v2
+
+[Container Linux][container-linux-docs] filesystem includes `etcd.service` for v0.x and `etcd2.service` for v2.x, both of which will be removed in later versions. See [Container Linux FAQ][container-linux-faq] for more detail.
+
+To check if it exists:
+
+```
+systemctl cat etcd.service
+systemctl cat etcd2.service
+```
+
+To check if it is running:
+
+```
+systemctl status etcd.service
+systemctl status etcd2.service
+```
+
+To stop and disable:
+
+```
+systemctl stop etcd.service
+systemctl stop etcd2.service
+
+systemctl disable etcd.service
+systemctl disable etcd2.service
+```
+
+## etcd v3
+
+[Container Linux][container-linux-docs] filesystem does not include etcd v3 binaries, thus different versions of etcd v3 can be fetched via `etcd-member.service`.
```

Comment:
[Container Linux][container-linux-docs] does not include etcd 3.x binaries by default. Different versions of etcd 3.x can be fetched via `etcd-member.service`.